index.js

```diff
@@ -103,7 +103,35 @@ const DEFAULT_METRIC_OPTIONS = {
   sendCallback: () => {},
   maxCapacity: 20
 };
-
+const UNITS = {
```

Comment:
Can you add a documentation link here to the CloudWatch docs for metric units (the one from the PR comment)?